src/main/java/util/ApplicationUtils.java

```diff
@@ -0,0 +1,22 @@
+package util;
+
+import org.commcare.api.persistence.SqlSandboxUtils;
+
+/**
+ * Utility methods for dealing with Applications
+ */
+public class ApplicationUtils {
+
+    public static Boolean deleteApplicationDbs(String appId) {
+        String dbPath = "dbs/" + appId;
```

Comment:
Hm, this exposes a potential problem. Until now all these dbs had been shared across users since the assumption was they'd basically remain unchanged, but if we start living in a world where they can be wiped out we'll need to introduce some locking around them or have namespaced per-user installs such that we can piggy back on the per-user locking (leaning towards the latter)